meta-mender-core/recipes-mender/mender-migrate-configuration/mender-migrate-configuration.bb

```diff
@@ -14,6 +14,16 @@ ALLOW_EMPTY_${PN} = "1"
 
 RDEPENDS_${PN} += "jq"
 
+DEPENDS += "mender"
+
+do_check_split_conf() {
+    if [ -f ${STAGING_DIR_TARGET}/data/mender/mender.conf ]; then
```

Comment:
`STAGING_DIR_TARGET`, aha!

Reply:
Yes. But the key was to add `SYSROOT_DIRS += "/data"` to the mender recipe so that it is available for other recipes